test/src/receiver/main.cpp

```diff
@@ -28,20 +28,20 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp : public cadmium::in_port<Message_t>{};
+struct inp : public cadmium::in_port<message_t>{};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp : public cadmium::out_port<Message_t>{};
+struct outp : public cadmium::out_port<message_t>{};
 
 
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
 template<typename T>
-class ApplicationGen : public iestream_input<Message_t,T> {
+class ApplicationGen : public iestream_input<message_t,T> {
 public:
   ApplicationGen() = default;
-  ApplicationGen(const char* file_path) : iestream_input<Message_t,T>(file_path) {}
+  ApplicationGen(const char* file_path) : iestream_input<message_t,T>(file_path) {}
 };
 
 
```

Comment:
Many lines in this file are more than 80 characters in length (example: line 60-67)

Reply:
Changes done. Thank You.